include/clap/ext/draft/track-info.h

```diff
@@ -4,24 +4,48 @@
 #include "../../color.h"
 #include "../../string-sizes.h"
 
+// This extensions let the plugin query info about the track it's in.
+// It is useful when the plugin is created, to initialize some parameters (mix, dry, wet)
+// and pick a suitable configuartion regarding audio port type and channel count.
+
 static CLAP_CONSTEXPR const char CLAP_EXT_TRACK_INFO[] = "clap.track-info.draft/0";
```

Comment:
Should be increased to .draft/1 now I think?

Reply:
Yes!